editor/modes/visual-editor/block-list.js

```diff
@@ -83,14 +83,15 @@ class VisualEditorBlockList extends Component {
 	onPointerMove( { clientY } ) {
 		const BUFFER = 20;
 		const { multiSelectedBlocks } = this.props;
-		const y = clientY + window.pageYOffset;
+		const boundaries = this.refs[ this.selectionAtStart ].getBoundingClientRect();
```

Comment:
Because `onPointerMove` will be called very frequently, could there be a performance implication to this call? Would it be possible to catch this in `onSelectionStart`. At least since the function is throttled, it may not be too much of an issue.

Reply:
Right, it will stay the same as long as none of the parent containers scroll, but we'd have to recalculate on scroll. I wanted to adjust that, but at the same time found another bug... The `scroll` event won't fire on `window` with the scroll container. It doesn't bubble, whereas `mousemove` does. 😔 Looking further.

Reply:
This would be a really ugly fix... https://github.com/jquery/jquery-ui/blob/1.12.1/ui/scroll-parent.js#L27

Reply:
Or we store a ref for the editor layout, then pass it down I guess...

Reply:
Okay, found a fix for that. Coming back to your comment, I'm not sure if it's a good idea to only recalculate the boundaries on scroll, exactly because it's throttled. What if there's a bunch of `scroll` events and then a bunch of `mousemove` events, and the one that passes through is a `mousemove` event? The boundaries won't be recalculated and will maybe be off by a little bit, maybe just enough to feel buggy?

Reply:
And since we'd allow it to fire every 250 ms on `scroll` anyway, I'm not concerned to let if fire on `mousemove` as well, especially since they're both part of the same throttle.